classes/PodsAdmin.php

```diff
@@ -1360,9 +1366,38 @@ public function admin_setup_edit_proto( $duplicate, $obj ) {
 
 		wp_localize_script( 'pods-dfv', 'podsAdminConfig', $config );
 
+		$this->admin_js_dfv_config();
+
 		pods_view( PODS_DIR . 'ui/admin/setup-edit-proto.php', compact( array_keys( get_defined_vars() ) ) );
 	}
 
+	/**
+	 * Handle setting up the JS DFV config.
+	 *
+	 * @since 2.8.0
+	 */
+	protected function admin_js_dfv_config() {
+		$config = [
+			'wp_locale'      => $GLOBALS['wp_locale'],
```

Comment:
:warning: Array double arrow not aligned correctly; expected 2 space(s) between "'wp_locale'" and double arrow, but found 6.

<sub>hash: 086693a2c49873554c0a493fb8b054e2</sub>